openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@openc3/cosmos-tool-bucketexplorer",
+  "version": "5.3.1-beta0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build --no-module",
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint",
+    "serve:standalone": "vue-cli-service serve --mode standalone",
+    "test:components": "vue-cli-service test:components"
+  },
+  "resolutions": {
+    "acorn": "npm:acorn-with-stage3",
```

Comment:
This line cost me three hours.  It keeps tools from building outside of the monorepo for some reason.  Do you know if we need these resolutions anymore?

Reply:
I don't know why that is there. I just build with our scripts.